### PR TITLE
Update maintainer emails

### PR DESCRIPTION
**Pull request by bsaul**
Also adds support for nix.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-                    email = "bsaul@novisci.com"),
+                    email = "bradleysaul@fastmail.com"),
              person("Alex", "Breskin", role = c("ctb"), 
-                    email = "abreskin@novisci.com"),
+                    email = "alexbreskin@gmail.com"),
              person("Catie", "Wiener", role = c("ctb"), 
-                    email = "cwiener@novisci.com"),
+                    email = "cwiener@live.unc.edu"),
              person("Matt", "Phelan", role = c("ctb"), 
                     email = "mphelan@novisci.com"))
```
Comment (bsaul) [inline DESCRIPTION]: Need Matt's contact. Is it the same?

File: DESCRIPTION [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 Type: Package
 Title: Compute Standardized Mean Differences
-Version: 0.6.6
+Version: 0.6.7
 Authors@R: c(person("Bradley", "Saul", role = c("aut", "cre"), 
-                    email = "bsaul@novisci.com"),
+                    email = "bradleysaul@fastmail.com"),
              person("Alex", "Breskin", role = c("ctb"), 
-                    email = "abreskin@novisci.com"),
+                    email = "alexbreskin@gmail.com"),
```
Comment (bsaul) [inline DESCRIPTION:8]: @asb2111 - is this email Ok for you?

File: DESCRIPTION [hunk 3]
```diff
@@ -27,7 +27,7 @@ Suggests:
   rmarkdown
 License: MIT + file LICENSE
 URL: https://docs.novisci.com/smd/
```
Comment (bsaul) [inline DESCRIPTION:29]: Be sure to update site.

**Comment by asb2111** [comment]
Yes, thank you!On Mar 12, 2024, at 1:05 PM, Bradley Saul ***@***.***> wrote:﻿
@bsaul commented on this pull request.



In DESCRIPTION:
>               person("Alex", "Breskin", role = c("ctb"), 
-                    email = ***@***.***"),
+                    email = ***@***.***"),

@asb2111 - is this email Ok for you?

**Pull request merged**